lib/sles4sap_publiccloud.pm

```diff
@@ -626,24 +665,24 @@ sub get_promoted_instance {
 sub wait_for_sync {
     my ($self, %args) = @_;
     my $timeout = bmwqemu::scale_timeout($args{timeout} // 900);
-    my $online_str = check_version('>=2.1.7', $self->pacemaker_version()) ? '[1-9]+' : 'online';
+    my $online_str = check_version('>=2.1.7', $self->pacemaker_version()) ? '4' : 'online';
```

Comment:
I know maybe it is already explained in other places, but `'4'` here maybe deserve a comment to reference what 4 is about.